src/create-schedule/components/ScheduleTagInput.tsx

```diff
@@ -8,7 +8,10 @@ const ScheduleTagInput = () => {
   const { tag } = useRecoilValue(scheduleAnswers);
 
   return (
-    <div className="w-[628px] border border-[#E0E0E0] rounded-[5px] px-[8px] mb-[20px] focus-within:border-[#4285F4]">
+    <div
+      id="태그"
+      className="w-[628px] border border-[#E0E0E0] rounded-[5px] px-[8px] mb-[20px] focus-within:border-[#4285F4]"
```

Comment:
```suggestion
      className="w-[628px] border border-[#E0E0E0] rounded-md px-2 mb-5  focus-within:border-[#4285F4]"
```

[Lv2] `E0E0E0` `4285F4` 둘다 자주 사용되고있습니다! 따로 분리해주세요!